adapter_pipelines/Optimus_mouse/adapter_example_static.json

```diff
@@ -0,0 +1,14 @@
+{
+  "AdapterOptimus.whitelist": "gs://hca-dcp-sc-pipelines-test-data/whitelists/737K-august-2016.txt",
+  "AdapterOptimus.tar_star_reference": "gs://hca-dcp-mint-test-data/20190507-PipelinesSurge/mouse_reference/star_primary_gencode_mouse_vM21.tar",
+  "AdapterOptimus.annotations_gtf": "gs://broad-dsde-mint-dev-teststorage/ajc-test/gencode.vM21.annotation.gtf.gz",
+  "AdapterOptimus.ref_genome_fasta": "gs://broad-dsde-mint-dev-teststorage/ajc-test/GRCm38.primary_assembly.genome.fa",
```

Comment:
These should be moved to a more proper place if possible, such as under `gs://hca-dcp-sc-pipelines-test-data`